lib/railway_ipc/core/commands_consumer.ex

```diff
@@ -30,7 +30,11 @@ defmodule RailwayIpc.Core.CommandsConsumer do
     ack_func.()
   end
 
-  def post_processing({:skip, %{result: %{reason: reason}, payload: payload}}, ack_func) do
+  def post_processing(
```

Comment:
oh no! a bug! (we need to define this function to take in a fourth arg of the `publish_func` b/c that is how it is being called.)